content/guides/kubernetes/harbor-gs.md

```diff
@@ -37,28 +38,26 @@ Before you get started, you’ll need to do the following:
 
 - **Install Helm 3**: Helm is used in this guide to install Contour and Harbor. A guide for installing Helm can be found [here](https://helm.sh/docs/intro/install/).
 
-- **Create a Kubernetes cluster**: This guide was built using Google Kubernetes Engine (GKE) with Kubernetes version 1.17. Any Kubernetes cluster with access to the internet should work fine, but your results may vary depending on the version you use. Initial testing with 1.16 resulted in errors during the Harbor install. For a guide on creating a GKE cluster, see [this page](https://cloud.google.com/kubernetes-engine/docs/quickstart) from Google. Ensure your [`kubectl` context](https://kubernetes.io/docs/reference/kubectl/cheatsheet/) is using this cluster. 
-
-- **Buy a domain name**: You will need a domain name that you control in order to configure DNS. This guide uses a Google-owned domain and DNS zone, but instructions can be modified for other providers.
+- **Create a Kubernetes cluster**: This guide was built using Google Kubernetes Engine (GKE) with Kubernetes version 1.17. Any Kubernetes cluster with access to the internet should work fine, but your results may vary depending on the version you use. Initial testing with 1.16 resulted in errors during the Harbor install. For a guide on creating a GKE cluster, see [this page](https://cloud.google.com/kubernetes-engine/docs/quickstart) from Google. Ensure your [`kubectl` context](https://kubernetes.io/docs/reference/kubectl/cheatsheet/) is using this cluster.
 
 - **Install watch**: watch is a small command-line utility that continually shows the output of a command being run. This allows you to monitor `kubectl get pods`, for example, without explicitly re-running the command multiple times. Instructions for installing on macOS are [here](https://osxdaily.com/2010/08/22/install-watch-command-on-os-x/).
 
 - **About 30 minutes**:  It could take more time than that; it will depend on how long the Let’s Encrypt servers take to issue certificates. But in most of my testing for writing this post, it took about 30 minutes.
 
+- *Optional - buy a domain name*: You can use `.xip.io` addresses to avoid needing to buy a domain. Otherwise you will need a domain name that you control in order to configure DNS. This guide uses a Google-managed domain and DNS zone, but instructions can be modified for other providers.
```

Comment:
Worth adding a link to http://xip.io/ for those who might not know what it is or how it works?

Reply:
good call, done!